files/helm/fastgpt/Chart.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v2
+name: fastgpt
```

Comment:
Maybe it will force replace fastgpt‘s docker image?
shaul we change to `fastgpt-helm`?

Reply:
The image tags are different. The helm chart image tag has the suffix `-helm`.

See also https://github.com/labring/FastGPT/pull/1023/files#diff-e55e8b28bf787ec17d6f2b235a396d3256da0ec8b9818603c61b46197d84e6c0R33